enterprise/utils.py

```diff
@@ -700,7 +700,7 @@ def get_enterprise_customer(uuid):
     EnterpriseCustomer = enterprise_customer_model()
     try:
         return EnterpriseCustomer.objects.get(uuid=uuid)
-    except EnterpriseCustomer.DoesNotExist:
+    except (EnterpriseCustomer.DoesNotExist, ValidationError):
```

Comment:
Why do we need `ValidationError`? seems it is extra.

Reply:
Incase of an invalid uuid ... it throws a validation error